### PR TITLE
Add config for autoenv

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,22 @@
+if [[ -d .venv ]]
```
Comment (cfconrad) [inline .env:1]:
```suggestion
if [[ -d .venv ]] && [[ -z "$VIRTUAL_ENV" ]]
```

File: .env [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+then
+    source ./.venv/bin/activate
```
Comment (asmorodskyi) [inline .env:3]: your upper suggestion , also meant ? 
```suggestion
    source "$VIRTUAL_ENV"
```

Comment (asmorodskyi) [reply .env:3]: I thought about that , but beauty of this thing exactly in a fact that you **don't need to do anything beside cd'ing to dir** , with variable approach I would need to type "VIRTUAL_ENV=./.venv/bin/activate" and than there is no big difference to type "source ./venv/bin/activate" :)

Comment (cfconrad) [reply .env:3]: hm, don't get it.
I just meant, you should not `source ./.venv/bin/activate` twice.

**Pull request merged**